src/actor/monster/styles/monster/_main-tab.scss

```diff
@@ -0,0 +1,42 @@
+#typicalTraits {
+  width: calc(100% - 87px);
+}
+
+#monsterExperienceFields {
+  flex-direction: row;
+}
+
+.monster-main-fields {
```

Comment:
I see we're switching between cases here. Is there a convention we want to follow?

Reply:
I meant typecase - you explained that classes are kebab, and ids are camel